sdk/tables/Azure.Data.Tables/src/Sas/TableSasBuilder.cs

```diff
@@ -256,7 +256,7 @@ public TableSasQueryParameters ToSasQueryParameters(TableSharedKeyCredential sha
         /// </returns>
         private static string GetCanonicalName(string account, string tableName) =>
             // Table: "/table/account/tablename"
-            string.Join("", new[] { "/table/", account, "/", tableName });
+            string.Join("", new[] { "/table/", account, "/", tableName.ToLowerInvariant() });
```

Comment:
Out of curiosity, why `string.Join` here instead of a bling string or `string.Format` ?

Reply:
Good question - I don't think there is a good reason except that when we build the string to sign, we join the parts and delimit by "\n", so this pattern stuck around here somehow.